field_manager/field_manager.F90

```diff
@@ -1193,7 +1193,7 @@ subroutine new_name ( list_name, method_name_in , val_name_in)
   if (val_name(1:1) .eq. squote) then  !{
 
     if (val_name(length:length) .eq. squote) then
-      val_name = val_name(2:length-1)
+      val_name = val_name(2:length-1)//repeat(" ",len(val_name)-length+2)
```

Comment:
This seems like a fancy way of saying ```//"  "```

Reply:
The repeat ensures that the de-quotation-marked string has 2 blank spaces appended to it to match original string size. `//" "` would only append 1 blank space. `//" "//" "` would achieve the same result, but is less explicit, and more error-prone (easy to accidentally add too many spaces in between quotes).